apps/relayer/tests/deploy.ts

```diff
@@ -1,5 +1,5 @@
 import hardhat from "hardhat";
-import { deploy, deployPoll, deployVkRegistryContract, joinPoll, setVerifyingKeys, signup } from "maci-cli";
+import { deploy, deployPoll, deployVkRegistryContract, joinPoll, setVerifyingKeysCli, signup } from "maci-cli";
```

Comment:
Maybe we can use `setVerifyingKeys` from `maci-sdk` for relayer?

Reply:
was thinking to do that separately, ideally we remove all imports from the cli and use only sdk functions. will work as part of the test refactoring